src/services/user.service.js

```diff
@@ -0,0 +1,21 @@
+import db from '../models';
+
+/**
+ * Class user sdervices
+ */
+class UserServices {
+  /**
+   *
+   * @param {string} email
+   * @returns {object} user
+   */
+  static async findByEmail(email) {
+    const user = await db.user.findOne({
+      where: { email }
+    });
+    if (!user) return null;
+    return user;
+  }
```

Comment:
It would be a good idea if this was in the users' controller.

Reply:
Okay, done. Thanks for that